python/demo/mixed-elasticity-sc/static-condensation-elasticity.py

```diff
@@ -20,8 +20,8 @@
 import ufl
 from dolfinx import geometry
 from dolfinx.cpp.fem import Form_complex128, Form_float64
```

Comment:
Is it possible to remove this and use the factory function?

Reply:
Maybe, but it's a bit separate form this change. Might be possible to so something neater soon.